src/features/query-bar/flowTypes.js

```diff
@@ -1,11 +1,13 @@
 // @flow
-import { AND, OR } from './constants';
+import { AND, OR, EMPTY_CONNECTOR } from './constants';
 
-export type SelectOptionType = {
+export type ConnectorType = typeof AND | typeof OR | typeof EMPTY_CONNECTOR;
+
+export type OptionType = {
     displayText: string,
-    fieldId?: string,
+    fieldId: string,
```

Comment:
The name `fieldId` seems to incorrectly borrow from Template.Field. Could it be "id"?

Reply:
Yes